src/index.html

```diff
@@ -1,14 +1,20 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page__body">
+    <div class="stopwatch stopwatch--position">
+      <div class="stopwatch__minute-hand">
+        <div class="stopwatch__center-hand">
+          <div class="second-hand"></div>
```

Comment:
```suggestion
          <div class="stopwatch__second-hand"></div>
```
And do you really need to nest them?

Reply:
Yes, I think it was right to put the hand in the middle. Since it is part of the center-hand.